### ISSUE TITLE
Response is truncated when using brotli compression

### ISSUE THREAD

**Issue by art-in** [actix/actix-web#1224]
Hi, I've noticed that response is truncated when serving files with brotli compression.  

Repro: https://github.com/art-in/actix-web-brotli-bug

main.rs
```rust
#[actix_rt::main]
async fn main() -> io::Result<()> {
    HttpServer::new(|| {
        App::new()
            .wrap(middleware::Compress::default())
            .service(fs::Files::new("/", "static").index_file("index.html"))
    })
    .bind(BIND_URL)?
    .start()
    .await
}
```

Cargo.toml
```toml
[dependencies]
actix-rt = "1.0.0"
actix-web = "2.0.0-alpha.6"
actix-files = "0.2.0-alpha.3"
```

Meanwhile, how can I opt-out brotli and use only gzip/deflate?

**Comment by fafhrd91** [comment]
brotli compression is fixed in actix-http 1.0.1
there is no way to disable brotli only

**Comment by art-in** [comment]
it works, thanks!
